adapters/handlers/grpc/v1/prepare_reply.go

```diff
@@ -327,12 +383,12 @@ func extractGroup(raw any, searchParams dto.GetParams, scheme schema.Schema, use
 		returnObjectsUntyped[i] = group.Hits[i]
 	}
 
-	objects, groupedGenerativeResults, err := extractObjectsToResults(returnObjectsUntyped, searchParams, scheme, true, usesMarshalling)
+	objects, _, err := extractObjectsToResults(returnObjectsUntyped, searchParams, scheme, true, usesMarshalling)
```

Comment:
why this change?

Reply:
The generative result in a group by query doesn't come from the `hits` field of the map but instead from the `generate` field on the same level as the `hits` field. Since `extractObjectsToResults` here is acting on `returnObjectsUntyped`, which is a list of `group.Hits` then it cannot have generated results in it